lib/davinci_dtr_test_kit/docs/dtr_smart_app_suite_description_v201.md

```diff
@@ -52,35 +52,48 @@ validated with the Java validator using `tx.fhir.org` as the terminology server.
 
 ### Quick Start
 
-Inferno does not currently include the ability to launch the client. Therefore, clients
-must be manually configured to point to Inferno's simulated server endpoints. The endpoints
-can be inferred from the URL of the test session which will be of the form `[URL prefix]/dtr_smart_app/[session id]`: (NOTE: both currently use the same URL)
-- Payer Server Base FHIR URL: `[URL prefix]/custom/dtr_smart_app/fhir`
-- Light EHR Base FHIR URL: `[URL prefix]/custom/dtr_smart_app/fhir`
-
-In order for Inferno to associate requests sent to locations under these base URLs with this session,
-it needs to know the bearer token that the app will send on requests, for which 
-there are two options.
-
-1. If you want to choose your own bearer token, then
-    1. Select the "2. Basic Workflows" test from the list on the left (or other target test). 
-    2. Click the '*Run All Tests*' button on the right.
-    3. In the "access_token" field, enter the bearer token that will be sent by the client 
-       under test (as part of the Authorization header - `Bearer <provided value>`).
-    4. Click the '*Submit*' button at the bottom of the dialog.
-2. If you want to use a client_id to obtain an access token, then
-    1. Click the '*Run All Tests*' button on the right.
-    2. Provide the client's registered id "client_id" field of the input (NOTE, Inferno doesn't support the
-        registration API, so this must be obtained from another system or configured manually).
-    3. Click the '*Submit*' button at the bottom of the dialog.
-    4. Make a token request that includes the specified client id to the
-        `[URL prefix]/custom/dtr_smart_app/mock_auth/token` endpoint to get
-        an access token to use on the request of the requests.
-
-In either case, the tests will continue from that point. Further executions of tests under
-this session will also use the selected bearer token.
-
-Note: authentication options for these tests have not been finalized and are subject to change.
+This test suite can be run in two modes, each described below:
+1. [EHR launch mode](#ehr-launch)
+2. [Standalone launch mode](#standalone-launch)
+
+At this time, Inferno's simulation of the payer server that provides the questionnaires
+uses the same base server url and access token and apps will need to be configured to
+connect to it as well. See [here](#combined-payer-and-ehr-fhir-servers) for details.
```

Comment:
This anchor link doesn't work, because the headings in this markdown don't get IDs

```suggestion
connect to it as well. See the "Combined payer and EHR FHIR servers" section below for details.
```